Tests/js/views/services/plugins/assets/ez-searchplugin-tests.js

```diff
@@ -926,8 +931,15 @@ YUI.add('ez-searchplugin-tests', function (Y) {
                     if (query === that.locationQuery) {
                         cb(false, that.locationResponse);
                     } else if (query === that.contentQuery) {
+                        var contentIds = [];
+
+                        Y.Array.each(
```

Comment:
it would be a bit more elegant to use use `map` in such case. Also there are small CS issues (`);` is badly placed, a space is missing after `function`)